sdk/core/azure/README.rst

```diff
@@ -1,84 +1,17 @@
 Microsoft Azure SDK for Python
 ==============================
 
-This is the Microsoft Azure bundle.
+This package was the Microsoft Azure bundle.
 
-This package does not contain any code in itself. It installs a set
-of packages that provide Microsoft Azure functionality.
+Starting v5.0.0, this package is deprecated. Please install the service specific packages prefixed by `azure` needed for your application.
```

Comment:
Link back to documentation listing packages? I believe we have such a page (maybe the GitHub.io page?). If we don't, we should.

Reply:
https://azure.github.io/azure-sdk/releases/latest/index.html#python-packages or https://aka.ms/azsdklist#python-packages

Reply:
This seem to only be the "track 2" libraries. Which is probably an issue that we should address.

Reply:
Click the "All Libraries" link to see them all 
![image](https://user-images.githubusercontent.com/2163001/78306119-76607b00-74f7-11ea-9262-dbb9370f0fc1.png)

Reply:
Ah, Missed that. Thanks!

Then I'd link to https://azure.github.io/azure-sdk/releases/latest/all/python.html (or, more specifically, whatever aka.ms link we have for that)

Reply:
I don't think there is a way from https://aka.ms/azsdklist to get the full list. @jongio thoughts?

Reply:
I'm working on getting an aka to point to the full list.  Give me a couple of days.

Reply:
Please use https://aka.ms/azsdk/python/all